.github/workflows/test.yaml

```diff
@@ -242,6 +245,53 @@ jobs:
         if: failure()
         run: bash .github/slack/notify-failure.sh ${{ github.job }} ${{ matrix.feature }}
 
+  # These tests are expected to fail
+  # Meaning a successfull run actually shows a non zero exit code in the logs
+  docker-failing:
+    needs: [detect_jobs_to_run]
+    if: ${{ fromJson(needs.detect_jobs_to_run.outputs.jobs)['docker'] == true }}
```

Comment:
Hm, does that follow the usual pattern?
Wouldn't we usually have a top level `docker-failing` folder instead?

This is complicated enough, let's not introduce more special cases if possible.

Reply:
That's a good point, done. I see a couple of errors in the CI, will quickly solve after dinner

Reply:
Fixed. Two optional tests are failing, which were failing before this PR (and my previous other PRs as well)

Reply:
Yep, can confirm those are expected.
(We might want to split out "optional" and "expected failing" tests some time...)